.config-schema.json

```diff
@@ -302,6 +302,12 @@
                     "type": "float",
                     "default": "0.2",
                     "x-env-variable": "OPENFGA_TRACE_SAMPLE_RATIO"
+                },
+                "serviceName": {
+                    "description": "the service name included in sampled traces.",
```

Comment:
Use this description in the help message in `cmd/run/flags.go`. They are currently different descriptions.

Reply:
done